packages/snap/package.json

```diff
@@ -18,14 +18,14 @@
     "build:clean": "yarn clean && yarn build",
     "build:website": "node ./scripts/build-website.js",
     "clean": "rimraf dist",
+    "dev": "mm-snap watch",
     "eval:snap": "mm-snap eval -b dist/bundle.js --verboseErrors",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path .gitignore",
     "proofPrep": "ts-node src/scripts/proofGenerationPrep.ts \"$@\"",
     "serve": "mm-snap serve",
-    "start": "mm-snap watch",
```

Comment:
If the start script is removed here, the instructions in the top level README do not work anymore. Can you please update the instructions or add this line back?
Should we also add the installation of turbo to the instructions or is it just an optional thing?

Reply:
"start" is returned